avocado/core/sysinfo.py

```diff
@@ -417,6 +427,16 @@ def __init__(self, basedir=None, log_packages=None, profiler=None):
             log.debug('File %s does not exist.', files_file)
             self.files = []
 
+        fail_files_file = config.get('sysinfo.collectibles.fail_files')
+        if os.path.isfile(fail_files_file):
+            log.info('Files configured by file: %s', fail_files_file)
+            self.fail_files = genio.read_all_lines(fail_files_file)
+            self.fail_files = [
+                files for files in self.fail_files if files not in self.files]
+        else:
+            log.debug('File %s does not exist.', fail_files_file)
+            self.fail_files = []
```

Comment:
This block of code looks pretty similar to the previous one. Maybe we should add this to a new private method and avoid duplicated lines?

Reply:
Let me try that

Reply:
@beraldoleal please check now